replication.go

```diff
@@ -1395,6 +1396,17 @@ func (cm *ContentManager) ensureStorage(ctx context.Context, content Content, do
 			done(time.Minute * 60)
 			return nil
 		}
+
+		bl, err := cm.FilClient.Balance(ctx)
+		if err != nil {
+			return errors.Wrap(err, "could not retrieve dataCap from client balance")
+		}
+
+		if bl.VerifiedClientBalance.LessThan(big.NewIntUnsigned(uint64(abi.UnpaddedPieceSize(content.Size).Padded()))) {
+			// how do we notify admin to top up datacap?
```

Comment:
A loud error log message *should* make it through. Ideally we get the error log rate on grafana down low enough that we can alert on it. 

aside from that, I think @lanzafame can help us set up some alerts